tools/check-templates

```diff
@@ -32,11 +32,11 @@ def check_our_files(modified_only, all_dups, targets):
             ftypes=['hbs', 'html'],
             group_by_ftype=True, exclude=EXCLUDED_FILES))
 
-    check_handlebar_templates(by_lang['hbs'])
-    check_html_templates(by_lang['html'], args.all_dups)
+    check_handlebar_templates(by_lang['hbs'], args.fix)
+    check_html_templates(by_lang['html'], args.all_dups, args.fix)
```

Comment:
Hey @whoodes  is there some reason we aren't using the parameter `fix` from function parameters over here?

Reply:
I was wondering the same with `args.all_dups` so I went with a consistent approach to how that parameter was being used. Is there a particular reason `all_dups` is passed via `args`, when it is being passed as a parameter in `check_our_files`?

Reply:
I think it was a bug in the previous code; it should be using the function parameter.

Reply:
@whoodes generally if you notice something seems off, you should comment on it, rather than just copying its convention.  Sometimes the existing convention is wrong :)